app/assets/javascripts/admin/addon/components/admin-config-area-sidebar-experiment.hbs

```diff
@@ -0,0 +1,20 @@
+<div
```

Comment:
This is all heavily experimental and I want to throw it away in future, so no need for translations etc.